### DISCUSSION TITLE
3.8.8 py build for new release 3.5 please

### DISCUSSION THREAD

**Discussion by quochuyonnet** [akai-katto/dandere2x#228] [General]
vulkan x4 support is great but my win7 os need 3.8py build please.

Best regards,

**Comment by akai-katto** [comment]
https://github.com/akai-katto/dandere2x/releases/tag/3.5

I added a main3.8.8.exe - let me know if it works.

**Reply by quochuyonnet** [reply]
> 
> 
> https://github.com/akai-katto/dandere2x/releases/tag/3.5
> 
> I added a main3.8.8.exe - let me know if it works.

Thank you alot akai-katto! It's make my day. I've tried it and it's work with Waifu2x-Converter-cpp but not with Vulkan/Caffe. I've attached some pics here for you.

_not working with vulkan/caffe:_
![caffe 2x single thread](https://user-images.githubusercontent.com/36237858/137901130-69802bf1-c963-444a-9abd-9babf9b01b4c.jpg)
![vulkan 2x single thread](https://user-images.githubusercontent.com/36237858/137901146-a619de45-1027-4e1f-9d6a-29cacd6a5a60.jpg)

![vulkan 4x multi thread 1](https://user-images.githubusercontent.com/36237858/137901189-faaf7c4d-e77c-4d43-a126-739a1d9476c0.jpg)
![vulkan 4x multi thread 2](https://user-images.githubusercontent.com/36237858/137901225-db2111fc-d66d-40f5-89ff-5d4b848df8a2.jpg)

Well done with converter_cpp:
![mediainfo](https://user-images.githubusercontent.com/36237858/137901235-c4989430-dd57-4fac-ba08-407646184d4c.jpg)
![waifu_converter_cpp](https://user-images.githubusercontent.com/36237858/137901242-0dc300e6-0723-46d0-84eb-71d14aa1f520.jpg)
e
